src/scripts/main.js

```diff
@@ -1 +1,38 @@
 'use strict';
+
+const form = document.querySelector('.contacts__form');
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--no-scroll');
+  } else {
+    document.body.classList.remove('page__body--no-scroll');
+  }
+});
+
+form.addEventListener('submit', event => {
+  event.preventDefault();
+  form.reset();
+});
+
+document.addEventListener('DOMContentLoaded', () => {
+  const chooseText = document.querySelector('.choose__text');
+  const creatorsText = document
+    .querySelector('.creators__main--desktop-paragraph');
```

Comment:
The querySelector should be placed on the same line as the assignment statement to improve readability.